aip/client-libraries/4234.md

```diff
@@ -0,0 +1,92 @@
+---
+id: 4234
+state: draft
+created: 2021-02-10
+---
+
+# Common service client mix-ins
+
+An API platform often has utility services that are shared amongst the discrete
```

Comment:
I am not sure that readers will intuitively know what "an API platform" means. It is not a concept that gets much discussion in AIPs.

Reply:
I reworded this to move away from this concept. Done.